doc/release-notes/5.7-release-notes.md

```diff
@@ -0,0 +1,109 @@
+
+Please watch the [online presentation](https://youtu.be/mWMPxCwB2ys?t=1794), [read the document with requirements](https://docs.google.com/document/d/1txdcFuxskRx_tLsDQ7KKLFTMR_r9IBhorDu3V_r445w/edit?ts=5fdbca9a) and join the [Dataverse Working Group on Ontologies and Controlled Vocabularies](https://dataverse.org/community-calls) if you have some questions and want to contribute.
+
+This functionality was initially developed by [Data Archiving and Networked Services](http://dans.knaw.nl/en/) (DANS-KNAW), the Netherlands, and funded by [SSHOC](https://sshopencloud.eu), "Social Sciences and Humanities Open Cloud". SSHOC has received funding from the European Union’s Horizon 2020 project call H2020-INFRAEOSC-04-2018, grant agreement #823782. It was further improved by the Global Dataverse Community Consortium (GDCC) and extended with the support of semantic search.
+
+## Major Use Cases
+
+Newly-supported major use cases in this release include:
+
+- Administrators will be able to set up integrations with external vocabulary services, allowing for autocomplete-assisted metadata entry, metadata standardization, and better integration with other systems. (Issue #7711, PR #7946)
+- Users viewing datasets in the root Dataverse collection will now see breadcrumbs that have have a link back to the root Dataverse collection (Issue #7527, PR #8078)
+- Users will be able to more easily differentiate between datasets and files through new iconography (Issue #7991, PR #8021)
+- Users retrieving large guestbooks over the API will experience fewer failures (Issue #8073, PR #8084)
+- Administrators will now be able to set a language for metadata exports at the document or element level (Issue #7388, PR #7958)
+- Administrators will now be able to specify the language of controlled vocabulary items (Issue #6751, PR #7959)
+- Administrators and curators can now receive notifications when a dataset is created (Issue #8069, PR #8070)
+- Administrators with large files in their installation can disable the automatic checksum verification process at publish time (Issue #8043, PR #8074)
+
+## Notes for Dataverse Installation Administrators
+
+### Dataset Creation Notifications for Administrators
+
+A new :SendNotificationOnDatasetCreation setting has been added. When true, administrators and curators (those who can publish the dataset) will get an email/notification when a new dataset is created. This makes it easier to track activity in a Dataverse and, for example, allow admins to follow up when users do not publish a new dataset within some period of time.
+
+### Skip Checksum Validation at Publish
```

Comment:
This should probably say "Skip Checksum Validation at Publish Based on Size"
(or something similar). There's always been a way to skip this validation; but in a less granular/configurable way.

Reply:
added in https://github.com/IQSS/dataverse/pull/8128/commits/c55a736450b8d1dee3e2356a560a668f3da3f2e4